eng/tools/spec-gen-sdk-runner/test/fixtures/specification/service1/data-plane/widget/tspconfig.yaml

```diff
@@ -1,6 +1,8 @@
 parameters:
   "service-dir":
     default: "sdk/widget"
+  "csharp-service-dir":
```

Comment:
@JoshLove-msft I guess you are doing this to scope this to just csharp but having a language specific parameter here defeats the purpose of having shared service-dir. If we have a plan to fix these then great but if not I would probably go with a model of setting `emitter-output-dir: {output-dir}/{service-dir}/{package-dir}` in the new section instead of creating a new parameter.